spec/requests/admin/affiliates_spec.rb

```diff
@@ -7,9 +7,25 @@
   let(:affiliate_user) { create(:affiliate_user) }
 
   before do
+    # Set up required configuration values for tests using environment variables
+    ENV["IFFY_TOKEN"] = "test_iffy_token"
+    ENV["OBFUSCATE_IDS_CIPHER_KEY"] = "test_cipher_key_32_characters_long"
+    ENV["OBFUSCATE_IDS_NUMERIC_CIPHER_KEY"] = "123456789"
```

Comment:
@d1dyup _a could you describe how adding these ENV vars here fixes the flakeyness in the context of this spec file?

Reply:
If that's the case, it should fail consistently, right?

I understand there’s flakiness with `requests/admin/affiliates_spec.rb`, but if missing ENV vars were the cause, we should be seeing these errors constantly or at least more frequently. For example, the most recent [build](https://github.com/antiwork/gumroad/actions/runs/17499348012) on main doesn't show this error.

Also, we actually inject these vars into the test environment in CI through Rails credentials [here](https://github.com/antiwork/gumroad/blob/9b8cccce299f6d70632d7091e68f8a2f953ecd94/.github/workflows/tests.yml/#L87-L98). It’s very unlikely that the flakiness is caused by missing ENV vars.